src/main/java/ssl/ois/timelog/service/activity/type/remove/RemoveActivityTypeUseCase.java

```diff
@@ -20,10 +21,12 @@ public RemoveActivityTypeUseCase(UserRepository userRepository) {
     public void execute(RemoveActivityTypeUseCaseInput input, RemoveActivityTypeUseCaseOutput output)
             throws DatabaseErrorException, ActivityTypeNotExistException, DuplicateActivityTypeException {
         User user = this.userRepository.findByUserID(input.getUserID());
+        ActivityType activityType = new ActivityType(input.getActivityTypeName(), input.getIsEnable(), input.getIsPrivate(), input.getIsDeleted());
 
-        user.deleteActivityType(input.getActivityTypeName());
+        user.deleteActivityType(input.getTargetActivityTypeName(), activityType);
```

Comment:
line 24 , 26 , 30 delete
 add 
  user.deleteActivityType(input.getActivityTypeName());
  to line 25